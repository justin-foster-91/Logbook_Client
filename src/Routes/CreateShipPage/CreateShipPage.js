import React from 'react';
import './CreateShipPage.css';

function CreateShipPage(props) {
  return (
    <div>
      <h1>
        Hello Create Ship Page
      </h1>
    </div>
  );
}

export default CreateShipPage;