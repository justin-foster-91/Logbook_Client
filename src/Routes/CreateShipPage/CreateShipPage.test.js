import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import CreateShipPage from './CreateShipPage';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<BrowserRouter><CreateShipPage /></BrowserRouter>, div);
  ReactDOM.unmountComponentAtNode(div);
})

it('renders the UI as expected', () => {
  const tree = renderer
    .create(<BrowserRouter><CreateShipPage /></BrowserRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();  
});
