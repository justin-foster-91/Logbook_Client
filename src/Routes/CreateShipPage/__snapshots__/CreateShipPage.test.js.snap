// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div>
  <h1>
    Hello Create Ship Page
  </h1>
</div>
`;
